chronos/transports/stream.nim

```diff
@@ -2115,6 +2218,50 @@ proc createStreamServer*(host: TransportAddress,
   createStreamServer(host, StreamCallback2(nil), flags, sock, backlog, bufferSize,
                      child, init, cast[pointer](udata), dualstack)
 
+proc createStreamServer*(port = Port(0),
```

Comment:
```suggestion
proc createStreamServer*(port: Port,
```

Maybe this makes sense, so that one cannot call `createStreamServer()` - when using `Port`, it is implicit that we want an IP socket (and not unix) - this way, `createStreamServer(Port(0))` is explicitly required to force the "type" of server

Reply:
Fixed in https://github.com/status-im/nim-chronos/pull/512/commits/5de7b1dd6362d9c93d270a4d7a074d804297c8c6